apps/api_accounts/lib/api_accounts/changeset.ex

```diff
@@ -288,6 +288,26 @@ defmodule ApiAccounts.Changeset do
     end
   end
 
+  @doc """
+  Ensure email address is downcase and trimmed.
+
+  ## Examples
+
+      format_email(changeset, :email)
+
+  """
+  @spec format_email(t, atom) :: t
+  def format_email(%Changeset{} = changeset, field) do
+    unformatted_email = Map.get(changeset.changes, field, "")
+
+    formatted_email =
+      unformatted_email
+      |> String.downcase()
+      |> String.trim()
+
+    put_change(changeset, :email, formatted_email)
```

Comment:
question: should this be `field` instead of `:email`?
```suggestion
    put_change(changeset, field, formatted_email)
```